infer/lib/rmvpe.py

```diff
@@ -2,6 +2,13 @@
 
 import numpy as np
 import torch
+try:

```

Comment:
Same, move to config.

Reply:
Removing this causes Torch not enabled with CUDA errors.

Reply:
OK.